recipe/embed-resources/main.go

```diff
@@ -12,15 +13,9 @@ func main() {
 	// the file server for rice. "app" is the folder where the files come from.
 	assetHandler := http.FileServer(rice.MustFindBox("app").HTTPBox())
 	// serves the index.html from rice
-	e.Get("/", func(c *echo.Context) error {
-		assetHandler.ServeHTTP(c.Response().Writer(), c.Request())
-		return nil
-	})
+	e.Get("/", standard.WrapHandler(assetHandler))
+
 	// servers other static files
-	e.Get("/static/*", func(c *echo.Context) error {
-		http.StripPrefix("/static/", assetHandler).
-			ServeHTTP(c.Response().Writer(), c.Request())
-		return nil
-	})
-	e.Run(":3000")
+	e.Get("/static/*", standard.WrapHandler(http.StripPrefix("/static/", assetHandler)))
```

Comment:
I'm still quite new to Go, but I thought the idea in the original code is to use `http.StripPrefix` to serve other static file requests that goes to `/static/*` from within the `Box`?

Sorry if I didn't get this right.